setup.py

```diff
@@ -3,8 +3,8 @@
 # TORCH_CUDA_ARCH_LIST
 #   specify which CUDA architectures to build for
 #
-# IGNORE_VER_ERR
-#   ignore version error for torch
+# IGN_TORCH_VER
```

Comment:
I think it's better to fully spell `IGNORE` for readability

Reply:
Fixed.